genesis/bin/genesis_playback.rs

```diff
@@ -166,13 +178,14 @@ fn main() {
                 .long("runtime-id")
                 .takes_value(true)
                 .required(true)
-                // TODO: Remove this default value to discourage use (see ekiden#1693).
-                .default_value("0000000000000000000000000000000000000000000000000000000000000000"),
+                // TODO: Remove this default value to discourage use (see oasis-core#1693).
+                .default_value("8000000000000000000000000000000000000000000000000000000000000000"),
```

Comment:
Can we just remove the default or will it break something (e.g., benchmarks)?

Reply:
Yeah, this can safely be removed. Benchmarks are currently broken anyway, so we'll add this flag where necessary.